File: Java/program/Mahasiswa.java
public class Mahasiswa extends SivitasAkademik
{
    private String nim;
    private String prodi;
    private String fakultas;

    Mahasiswa(){}

    Mahasiswa(String nik, String nama, String jenkel, String asalUniv, String emailEdu, String nim, String prodi, String fakultas)
    {
        this.nik = nik;
        this.nama = nama;
        this.jenkel = jenkel;
        this.asalUniv = asalUniv;
        this.emailEdu = emailEdu;
        this.nim = nim;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }
    
    public String getNIM()
    {
        return this.nim;
    }

    public void setNIM(String nim)
    {
        this.nim = nim;
    }

    public String getProdi()
    {
        return this.prodi;
    }

    public void setProdi(String prodi)
    {
        this.prodi = prodi;
    }

    public String getFakultas()
    {
        return this.fakultas;
    }

    public void setFakultas(String fakultas)
    {
        this.fakultas = fakultas;
    }
}

File: Python/program/Mahasiswa.py
from SivitasAkademik import SivitasAkademik

class Mahasiswa(SivitasAkademik):

    __nim = ""
    __prodi = ""
    __fakultas = ""

    def __init__(self, nik = "", nama = "", jenkel = "", asalUniv = "", emailEdu = "", nim = "", prodi = "", fakultas = ""):
        super().__init__(nik, nama, jenkel, asalUniv, emailEdu)
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas
    
    def getNIM(self):
        return self.__nim
    
    def setNIM(self, nim):
        self.__nim = nim
    
    def getProdi(self):
        return self.__prodi
    
    def setProdi(self, prodi):
        self.__prodi = prodi

    def getFakultas(self):
        return self.__fakultas
    
    def setFakultas(self, fakultas):
        self.__fakultas = fakultas